packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -263,14 +274,29 @@ class LinkViewController extends PlatformViewController {
   }
 
   @override
-  Future<void> dispose() async {
+  Future<void> dispose() {
+    context = null;
```

Comment:
```suggestion
    _isDisposed = true;
```

Reply:
I kind of like that the context being nullable (despite the $billion mistake), since it makes people aware that it may not be safe to access it.

Then I went over it again and realized the context wasn't actually used anywhere, so I removed it. I also realized that the element need not be late initialized, so there'd be no need for _isInitialized and the asserts, and so with nothing nullable, I added an _isDisposed as you suggest 😄